NuGet.Config

```diff
@@ -9,4 +9,8 @@
     <add key="AspNetRelease" value="https://www.myget.org/F/aspnetrelease/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
+  <disabledPackageSources>
```

Comment:
Any specific reason for disabling these? aspnetrelease is important once Microsoft branches off to the release branch (before a release). For example, for a couple weeks before beta8 was released, aspnetrelease contained the latest beta8, and aspnetcidev contained the latest rc1. In this state we want to sync up with beta8 and not rc1.